contributors/devel/conformance-guidelines.md

```diff
@@ -0,0 +1,83 @@
+# Conformance Test Guidelines
+
+The Kubernetes conformance test suite is a set of testcases, currently a
+subset of the integration/e2e tests, that the Architecture SIG has approved
+to define the core set of interoperable features that all Kubernetes
+deployments must support.
+
+Conformance tests in the default profile should cover features that are:
+
+- GA (not alpha or beta APIs, nor deprecated features)
+- portable (not dependent on provider-specific capabilities or on the public internet)
+- not an optional feature (e.g. not policy enforcement)
+- non-privileged (neither root on nodes, network, nor cluster)
```

Comment:
should work with default settings for all configuration parameters (example the default list of admission plugins should not have to be tweaked for passing conformance).

Reply:
Also tests that are flaky should not be added to the conformance suite.